academy/2-cosmos-concepts/12-testing.md

```diff
@@ -173,10 +173,10 @@ For instance, when unit-testing the `QueryCanPlayMoveRequest` handling, you coul
 
 ***Unit testing with mocks***
 
-If you need mocking another module's keeper, with the help of [`gomock`](https://github.com/golang/mock), you would:
+If you need to mock another module's keeper, with the help of [`gomock`](https://github.com/golang/mock) you would:
 
-* Create once and for all the mock types of your expected keeper interfaces, which includes the bank keeper.
-* Then for each test, create a mock controller, which acts as the puppeteer of your mocked bank keeper:
+* Create mock types for **all** of your expected keeper interfaces, which includes the bank keeper.
```

Comment:
What I want to convey here is that the mock types have to be created. Only once because after they are there. Unless the "expected keeper interfaces" have changed, in which case, the mock types have to be recreated.

Reply:
Corrected following our conversation.